src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/IllegalTokenCheck.xml

```diff
@@ -15,7 +15,10 @@
  native methods in Enterprise JavaBeans components.
  &lt;/p&gt;</description>
          <properties>
-            <property default-value="LABELED_STAT" name="tokens" type="anyTokenTypesSet">
+            <property default-value="LABELED_STAT"
+                      name="tokens"
+                      type="java.lang.String[]"
+                      validation-type="tokenSet">
```

Comment:
Suspicious

Reply:
https://checkstyle.org/checks/coding/illegaltoken.html#tokens

Reply:
Same we have in others
https://github.com/checkstyle/checkstyle/blob/b665b9dec38971beddf62a6650ab295b10753297/src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/IllegalTypeCheck.xml#L93-L98